### PR TITLE
Add entries filter for lead confidentiality and lead priority

### PR DESCRIPTION
**Pull request by frozenhelium**
- [x] Lead priority
- [x] Lead confidentiality
- [x] Lead published on

### CODE REVIEW

File: apps/entry/filter_set.py [hunk 1]
```diff
@@ -187,6 +209,28 @@ def get_filtered_entries(user, queries):
...
+    lead_confidentiality = queries.get('lead_confidentiality')
+    if lead_confidentiality:
+        entries = entries.filter(lead__confidentiality__in=lead_confidentiality)
+
+    lead_published_on_lt = queries.get('lead_published_on__lt')
+    print('>>>>>>>>>>>>>>>>>>>>>>>>>', lead_published_on_lt)
+    # if lead_published_on_lt:
+    #     entries = entries.filter(
+    #         lead__published_on__lt=datetime.date(lead_published_on_lt),
+    #     )
```
Comment (tnagorra) [inline apps/entry/filter_set.py]: Maybe we can remove these?

File: apps/entry/filter_set.py [hunk 2]
```diff
@@ -187,6 +209,28 @@ def get_filtered_entries(user, queries):
...
+
+    lead_published_on_gte = queries.get('lead_published_on__gte')
+    print('>>>>>>>>>>>>>>>>>>>>>>>>>', lead_published_on_gte)
+    # if lead_published_on_gte:
+    #     entries = entries.filter(
+    #         lead__published_on__gte=datetime.date(lead_published_on_gte),
+    #     )
```
Comment (tnagorra) [inline apps/entry/filter_set.py]: Maybe we can remove these?

Comment (frozenhelium) [reply apps/entry/filter_set.py]: not yet

File: apps/entry/filter_set.py (hunk 1 above)
Comment (frozenhelium) [reply apps/entry/filter_set.py]: not yet

**Review by thenav56 — APPROVED** [review]

**Pull request merged**